content/docs/custom-plugins/index.md

```diff
@@ -0,0 +1,51 @@
+---
+title: Installing Custom Plugins
+lastUpdated: '2021-03-16T17:55:00.0Z'
+description: How to add a custom Backstage plugin to Roadie
+---
+
+# Introduction
+
+Your plugins can be installed via npm or yarn, like publishing a normal package. We will provide you a unique scope for
+your plugins, and we host a dedicated npm registry which you can publish to.
+
+## Prerequisites
+
+- For our alpha release you'll need to give us some information upfront about your plugin. We plan to make all of this
+  self-service going forward but currently backstage doesn't support this out-of-the-box. For now, we will need you to
+  fill out [this form][form].
+
+- Get your credentials for the npm registry. Follow [this link][forgot-password] to reset your password. The username is
```

Comment:
Can we split out separate steps please, like the headings in the existing [Sentry docs](https://deploy-preview-137--roadie.netlify.app/docs/integrations/sentry/).

It looks like we have steps for

1. Get credentials
2. Configure NPM
3. Publish

Separate headings have the 2 advantages:

1. They are linkable so it's easier to send someone to a specific spot in the docs
2. We can automatically extract a table of contents in the future and make a sidebar on the right side of the page for easier navigation.

Reply:
:+1: good point. Done